src/adapter.js

```diff
@@ -0,0 +1,27 @@
+// API Blueprint parser for Fury.js
+
+import deckardcain from 'deckardcain';
+import protagonist from 'protagonist';
+
+export const name = 'api-blueprint';
+export const mediaTypes = [
+  'text/vnd.apiblueprint',
+];
+
+export function detect(source) {
+  return mediaTypes.indexOf(deckardcain.identify(source)) !== -1;
+}
+
+/*
+ * Parse an API Blueprint into refract elements.
+ */
+export function parse({source, generateSourceMap}, done) {
```

Comment:
Why are we not using the `exportSourceMap` option name? Why reverse the behavior and change the option name?